.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+*Preferable*
```

Comment:
why the inconsistent line breaks? OCD. :)

Reply:
I was trying to present an either/or type of option for reproduction - either a code sample or a list of steps, with preference (from a development perspective) being a code sample, but that a step-by-step reproduction is also an acceptable option.

I'm not a writer by any stretch, technical or otherwise, so that's what my developer's brain came up with to say in a fill-in-the-blanks manner "How do you reproduce this? If you can, please provide a gist/link/fiddle/etc. with some code. BUT, if that's not an option, no worries, you can provide a step-by-step process to reproduce as well."

So the "*Preferable*" here and "*Alternatively*" below were to be terse, which is... somewhat of a challenge for me in written formats! I had also started from the GitHub templates, and was trying to adapt them as much as possible to the needs and purposes of this project.

Reply:
AH that makes sense. So it's essentially a pick-list, not a "everything." :) Thanks! all good as-is, then. I was taught to be picky when i worked in the content team. :)